Moonshot/baddies/characters/MotherHen.gd

```diff
@@ -32,4 +32,4 @@ func on_hit(instance_id, damage) -> void:
 		attributes._on_hit(damage, global_position)
 
 func on_end() -> void:
-	call_deferred("free")
+	attributes.set_properties({"should_damage_on_collision": false})
```

Comment:
How come, do we want to be able to stand on top of it?

Reply:
Yes, want to be able to stand on it when dead, and then we can maybe inject it with the syringe. And when that happens, maybe it then destroys, and reveals the elevator shaft you you spoke about with Bill?